account_banking_pain_base/models/account_payment_order.py

```diff
@@ -519,6 +519,15 @@ def generate_party_acc_number(
             party_account_other = etree.SubElement(party_account_id, "Othr")
             party_account_other_id = etree.SubElement(party_account_other, "Id")
             party_account_other_id.text = partner_bank.sanitized_acc_number
+
+        if self.payment_type == "outbound" and party_type == "Dbtr":
+            currency = etree.SubElement(party_account, "Ccy")
+            currency.text = (
+                self.journal_id.currency_id
+                and self.journal_id.currency_id.name
+                or self.company_currency_id.name
```

Comment:
```suggestion
                self.journal_id.currency_id.name
                or self.company_currency_id.name
```

Or it should be the currency of each move line?

Reply:
@pedrobaeza 
PR adapted with your suggestion.